dev/puppetserver.conf.sample

```diff
@@ -5,7 +5,7 @@ global: {
 product: {
     update-server-url: "http://localhost/"
     name: {group-id: puppetlabs.dev
-           artifact-id: puppet-server}
+           artifact-id: puppetserver}
```

Comment:
This seems like maybe not a thing we should change without talking to BizDev? I know this file is not the actual thing that gets shipped, but I think we are testing this from it.